src/dotnet/commands/dotnet-add/dotnet-add-package/Program.cs

```diff
@@ -94,7 +100,7 @@ public override int Run(string fileOrDirectory)
                 GetProjectDependencyGraph(projectFilePath, tempDgFilePath);
             }
 
-            var result = NuGetCommand.Run(TransformArgs(RemainingArguments.First(), tempDgFilePath, projectFilePath));
+            var result = NuGetCommand.Run(TransformArgs(_packageNameArgument.Value, tempDgFilePath, projectFilePath));
```

Comment:
Good point.

I think we need three tests here though: 

one for passing dotnet add package blah --version blah1
one for dotnet add package blah blah blah (should fail)
and one for dotnet add package (should fail)